laskyawm.css

```diff
@@ -31,8 +31,8 @@ body:before {
 	position: absolute !important;
 	top: 25px;
 	left: 100px;
-	box-shadow: 0px 0px 20px rgba(0, 0, 0, .3);
 	background-color: rgba(255, 255, 255, .2);
+	border: 1px solid transparent;
```

Comment:
I have to admit the white border looks pretty cool. And I also agree that the black box-shadow doesn't fit very well with the light theme. However, without it it's kinda hard to see where one window ends and another starts, especially when everything is white (Firetext is white / light grey, Marketplace is white, border is now white). One idea would be to make the border the color of the (window) background, e.g. `.window .tabs { border: 1px solid rgba(255, 255, 255, .5); border-top: 0; }` but it doesn't work that well in Chrome. Other ideas?